ajax/libs/multi-level-push-menu/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "multi-level-push-menu",
+  "filename": "jquery.multilevelpushmenu.min.js",
+  "author": "adgsm",
+  "license": "MIT",
+  "homepage": "http://multi-level-push-menu.make.rs",
+  "description": "Cross-browser compatible jQuery plug-in allowing endless nesting of navigation elements.",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/adgsm/multi-level-push-menu.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.multilevelpushmenu.css",
```

Comment:
Let's use `jquery.multilevelpushmenu*`. For more details, you can refer https://github.com/isaacs/node-glob